hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/id/EdgeId.java

```diff
@@ -49,15 +49,14 @@ public class EdgeId implements Id {
             HugeKeys.OTHER_VERTEX
     };
 
-    private final Id ownerVertexId;
-    private final Directions direction;
-    private final Id edgeLabelId;
-    private final Id subLabelId;
-    private final String sortValues;
-    private final Id otherVertexId;
-
-    private final boolean directed;
-    private String cache;
+    protected final Id ownerVertexId;
+    protected final Id edgeLabelId;
+    protected final Id subLabelId;
+    protected final Id otherVertexId;
+    protected final Directions direction;
+    protected final boolean directed;
+    protected String sortValues;
+    protected String cache;
```

Comment:
better keep the order

Reply:
some son classes need to modify and release on-heap vars